app/assets/linux/SceneBuilder.desktop

```diff
@@ -5,6 +5,6 @@ Exec=APPLICATION_LAUNCHER
 Icon=APPLICATION_ICON
 Terminal=false
 Type=Application
-Categories=DEPLOY_BUNDLE_CATEGORY
+Categories=Development;Building;GUIDesigner;Java;
```

Comment:
This is one way to go, but since we have placeholders for mostly all the entries that are resolved by `jpackage`, the other option is to add the `--linux-menu-group` option to the `linux.yml` workflow. See jpackage [docs](https://docs.oracle.com/en/java/javase/17/jpackage/manage-installation.html#GUID-832F848E-E074-476C-A203-F081D0407081).